src/main/java/ch/njol/skript/expressions/ExprPlayerViewDistance.java

```diff
@@ -74,40 +68,55 @@ public Class<?>[] acceptChange(ChangeMode mode) {
 		}
 		return null;
 	}
-	
+
 	@Override
-	public void change(Event e, @Nullable Object[] delta, ChangeMode mode) {
+	public void change(Event event, @Nullable Object[] delta, ChangeMode mode) {
 		int distance = delta == null ? 0 : ((Number) delta[0]).intValue();
-		switch (mode) {
-			case DELETE:
-			case SET:
-				for (Player player : getExpr().getArray(e))
-					player.setViewDistance(distance);
-				break;
-			case ADD:
-				for (Player player : getExpr().getArray(e))
-					player.setViewDistance(player.getViewDistance() + distance);
-				break;
-			case REMOVE:
-				for (Player player : getExpr().getArray(e))
-					player.setViewDistance(player.getViewDistance() - distance);
-				break;
-			case RESET:
-				for (Player player : getExpr().getArray(e))
-					player.setViewDistance(Bukkit.getServer().getViewDistance());
-			default:
-				assert false;
+		for (Player player : getExpr().getArray(event)) {
```

Comment:
In this case I think we can avoid the switch statement inside of the loop :)
It probably doesn't matter that much performance wise, but you could have the set/get methods take in an array of players (just directly passed from Expression#getArray)